command/all_nfts.go

```diff
@@ -0,0 +1,45 @@
+package command
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hex337/alex-koin-go/config"
+	"github.com/hex337/alex-koin-go/model"
+)
+
+type AllNftsCommand struct{}
```

Comment:
you can create type functions instead of empty structs

Reply:
Can or should? What's the difference and can you write up an example? Maybe do that as part of a separate PR?